src/runner/api/itemRunner.js

```diff
@@ -392,6 +392,19 @@ const itemRunnerFactory = function itemRunnerFactory(providerName, data = {}, op
             return data;
         },
 
+        /**
+         * Replaces item data in rendered item
+         * @param {object} itemData
+         * @returns {itemRunner}
+         */
+        setData(itemData) {
+            data = itemData;
+            if (!closed && typeof provider.setData === 'function') {
```

Comment:
Why is `!closed` check needed? I think 'closed' state is just a half-transparent overlay, so images will still be loaded under it.

Reply:
My understanding is that `closed` is a permanent "locked" state. However, your comment makes me rethink if `setData` should still be allowed to be called here - could be a good idea if it prevents errors.

https://www.imsglobal.org/question/qtiv2p2p2/QTIv2p2p2-ASI-InformationModelv1p0/imsqtiv2p2p2_asi_v1p0_InfoModelv1p0.html#Lifecycle

Reply:
You're correct, this is the way we have implemented it in the QTI Item Runner (closed is a final state, you can't unclose it). But we can keep the API open for any kind of implementation.
As you can see https://github.com/oat-sa/tao-item-runner-fe/pull/32/files#diff-4dee0be6c9535c2234e23b85abd2283535a62ccb5fe601d0beb1f96b0a2ab14cR480 we keep the closed state resumable from the API, not from the implementation. 
This is definitely questionable, but it should be consistent.